pkg/api/types.go

```diff
@@ -1911,6 +1909,8 @@ type PodSpec struct {
 	// If not specified, the pod will be dispatched by default scheduler.
 	// +optional
 	SchedulerName string
+	// List of initialization containers belonging to the pod.
+	InitContainers []Container
```

Comment:
We don't sort based on id, and ids aren't added by sort.  Order of fields is supposed to be human / use first, machine last.  Move these back.